src/kernel-install/kernel-install.c

```diff
@@ -1257,8 +1259,8 @@ static int verb_remove(int argc, char *argv[], void *userdata) {
         if (arg_root)
                 return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "'remove' does not support --root= or --image=.");
 
-        if (argc > 2)
-                log_debug("Too many arguments specified. 'kernel-install remove' takes only kernel version. "
+        if (argc > 3)
```

Comment:
this doesn't look right, it's a named parameter not a positional one being added